include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -2024,16 +2029,26 @@ class FEEvaluation : public FEEvaluationAccess<dim,n_components_,Number>
   quadrature_point (const unsigned int q_point) const;
 
   /**
-   * The number of scalar degrees of freedom on the cell. Usually close to
-   * tensor_dofs_per_cell, but depends on the actual element selected and is
-   * thus not static.
+   * The number of degrees of freedom of a single component on the cell for
+   * the underlying evaluation object. Usually close to
+   * static_dofs_per_component, but the number depends on the actual element
+   * selected and is thus not static.
+   */
+  const unsigned int dofs_per_component;
+
+  /**
+   * The number of degrees of freedom on the cell accumulated over all
+   * components in the current evaluation object. Usually close to
+   * static_dofs_per_component*n_components=static_dofs_per_cell, but the
```

Comment:
maybe `static_dofs_per_cell=static_dofs_per_component*n_components` to ease reading?